services/app-web/src/constants/routes.ts

```diff
@@ -33,6 +34,7 @@ const RoutesRecord: Record<RouteT, string> = {
     ROOT: '/',
     AUTH: '/auth',
     DASHBOARD: '/dashboard',
+    GRAPHQL_EXPLORER: '/graphql-explorer',
```

Comment:
Let’s prefix this with `/dev/` in case we think of more things we want to build out for developers in the future